# eslint-plugin-template

## Rule Details

Describe the rule in detail.

Examples of **incorrect** code for this rule:

```js
const value = "hello world";
```

Examples of **correct** code for this rule:

```js
const value = "Hello World";
```

## Further Reading

- https://eslint.org/
